arm/Microsoft.ApiManagement/serviceResources/products/readme.md

```diff
@@ -57,13 +57,13 @@ Product groups list.
 
 ## Outputs
 
-| Output Name | Type |
-| :-- | :-- |
-| `productApisResourceIds` | array |
-| `productGroupsResourceIds` | array |
-| `productResourceGroup` | string |
-| `productResourceId` | string |
-| `productResourceName` | string |
+| Output Name | Type | Description |
+| :-- | :-- | :-- |
+| `productApisResourceIds` | array |  |
+| `productGroupsResourceIds` | array |  |
```

Comment:
Missing the description that was added.

Reply:
This seems do be an issue of the bicep build command. Check the resulting ARM template out: 

![image](https://user-images.githubusercontent.com/5365358/140608680-16b3b503-0524-43f8-841c-8c5bcac314ca.png)

Will create an issue in the bicep repo

Reply:
Updated the md too. However, unless this is fixed in the compiler, the readme will be overwritten again

Reply:
Opened an issue here: https://github.com/Azure/bicep/issues/5099